terraform/deployments/govuk-publishing-platform/signon_bearer_tokens.tf

```diff
@@ -45,7 +36,7 @@ module "publisher_to_publishing_api_bearer_token" {
 
   additional_tags                 = local.additional_tags
   api_user_email                  = local.signon_api_user.publisher
-  app_name                        = local.signon_app.publishing_api
+  app_name                        = local.signon_app.publishing_api.name
```

Comment:
Would it make sense to generate all these module instantiations with module for_each?

Reply:
Yes, good idea - if it's okay I've stuck in a TODO to follow up and do this in a later commit.